### PR TITLE
Various manifest fixes

### PR DESCRIPTION
**Pull request by hfiguiere**
- Fix the use of prefix for ffmpeg: the build produced is identical. See https://github.com/flathub/flatpak-builder-lint/issues/53
- Remove unecessary dbus for gvfs. See https://github.com/flathub/flathub/issues/2180#issuecomment-811984901
- Remove the branch for tracker. There already was a commit. Branch + commit will fail.

The build might still not pass the linter.

### CODE REVIEW

**Comment by hfiguiere** [comment]
The failure `finish-args-redundant-device-all` should be fixed with https://github.com/flathub/flatpak-builder-lint/pull/75

File: org.gnome.Totem.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
         /* Access DVDs */
         "--filesystem=/run/media",
         /* Browse gvfs */
-        "--talk-name=org.gtk.vfs", "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gtk.vfs.*",
```
Comment (hadess) [inline org.gnome.Totem.json:30]: This change is already in https://github.com/flathub/org.gnome.Totem.Devel/pull/7

File: org.gnome.Totem.json [hunk 2]
```diff
@@ -199,7 +199,7 @@
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/tracker.git",
-                    "branch": "tracker-3.2",
+                    /* branch is tracker-3.2 */
```
Comment (hadess) [inline org.gnome.Totem.json:202]: Why is this needed?

Comment (hfiguiere) [reply org.gnome.Totem.json:202]: The comment isn't, I left if as information.

Removing the `branch` is, because adding a commit to the branch will cause the build to fail. That's also why the linter doesn't allow branches anyway.

**Comment by hfiguiere** [comment]
bot, build

**Comment by hfiguiere** [comment]
looks like the linter changes have been deployed.

**Comment by hadess** [comment]
> Fix the use of prefix for ffmpeg: the build produced is identical. See https://github.com/flathub/flatpak-builder-lint/issues/53

Landed this in https://github.com/flathub/org.gnome.Totem.Devel/pull/7

> Remove unecessary dbus for gvfs. See https://github.com/flathub/flathub/issues/2180#issuecomment-811984901

I had my own version of this patch in https://github.com/flathub/org.gnome.Totem.Devel/pull/7

> Remove the branch for tracker. There already was a commit. Branch + commit will fail.

https://github.com/flathub/org.gnome.Totem.Devel/pull/7 didn't fail, and my local builds didn't fail either, so I'm not going to pick it up.

Thanks for the linter changes you made, much appreciated.
